easybuild/easyconfigs/a/awscli/awscli-1.17.7-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'awscli'
+version = '1.17.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
since both `Python` and `PyYAML` are in `GCCcore` now, this can also be moved down
```python
toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```